VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
It doesn't make sense to have `layout_height` set to `match_parent` in a vertical LinearLayout.

`id/notificaton_setting_row` has the same problem.